Documentation/DataSources/DeprecatedWorkbookRetrievalTool.md

```diff
@@ -0,0 +1,352 @@
+# Deprecated Workbook Retrieval Tool
```

Comment:
>Deprecated [](http://example.com/codeflow?start=2&length=10)

why is this in the datasources folder? why not have it in the same folder as the deprecation docs are? (either one)